gpkit/nomials/variables.py

```diff
@@ -148,6 +148,8 @@ def __new__(cls, shape, *args, **descr):
             i = it.multi_index
             it.iternext()
             descr.update({"idx": i})
+            assert value_option is not None
+            # TODO refactor logic for value_option and remove above assert
```

Comment:
What value does value_option take on if the 'if' statement in the loop never returns true. Is the default now None or 'sp_init'?